src/TypingIndicator.tsx

```diff
@@ -8,9 +9,9 @@ interface Props {
 }
 
 const TypingIndicator = (props: Props) => {
-  const [yCoords, setYCoords] = useState(new Animated.Value(200))
-  const [heightScale, setHeightScale] = useState(new Animated.Value(0))
-  const [marginScale, setmarginScale] = useState(new Animated.Value(0))
+  const [yCoords] = useState(new Animated.Value(200))
```

Comment:
Why?

Reply:
It's unused, why keep it and let the TS compiler complain about unused values?